### DISCUSSION TITLE
error certificate not valid

### DISCUSSION THREAD

**Discussion by farrostaufiqur** [Ljzd-PRO/KToolBox#15] [Q&A]
multiple error like this one:

Download failed - {'filename': '97119e25-9e54-4aea-a44a-087fe2e4970a.jpg', 'exception': SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'c3.kemono.party'. (_ssl.c:1131)")

query : python -m ktoolbox sync-creator https://kemono.su/fanbox/user/1709432/

**Comment by Ljzd-PRO** [comment]
Maybe caused by proxy?

**Reply by farrostaufiqur** [reply]
Idk, I just using it directly with my wifi. I think coom-dl also have same problem, maybe because there is some change with kemono server

**Comment by Ljzd-PRO** [comment]
If certificate on Kemono server expired, we may needed to add an option to bypass SSL certificate verification.

**Reply by farrostaufiqur** [reply]
Hope u add that option

**Comment by Ljzd-PRO** [comment]
Now can set`ssl_verify` option to `False` to disable SSL certificate verification.

https://github.com/Ljzd-PRO/KToolBox#configuration
